spec/es_helper.rb

```diff
@@ -29,13 +29,7 @@ def self.get_client(options)
   end
 
   def self.doc_type
-    if ESHelper.es_version_satisfies?(">=8")
-      nil
-    elsif ESHelper.es_version_satisfies?(">=7")
-      "_doc"
-    else
-      "doc"
-    end
+    "_doc"
```

Comment:
Should we remove es_version_satisfies as well?

Reply:
ack